core/storage/base_model/gae_models.py

```diff
@@ -419,6 +419,13 @@ class BaseCommitLogEntryModel(BaseModel):
     # The version number of the model after this commit.
     version = datastore_services.IntegerProperty()
 
+    @staticmethod
+    def get_deletion_policy():
```

Comment:
Will this be true for base classes, too? Should we prohibit overwriting of this method?

Reply:
It some it can be different (collection and exploration), thus we keep the possibility of overwriting.